src/Microsoft.DotNet.XHarness.iOS.Shared/Hardware/SimulatorSelector.cs

```diff
@@ -67,4 +67,20 @@ public ISimulatorDevice SelectSimulator(IEnumerable<ISimulatorDevice> simulators
         // Put Booted/Booting in front of Shutdown/Unknown
         return simulators.OrderByDescending(s => s.State).First();
     }
+
+    string GetiOSDeviceType(TestTargetOs testTargetOs, bool minVersion)
+    {
+        //default to version 16? 
+        var iOSversion = new Version(16, 0);
```

Comment:
Can the default come from https://github.com/dotnet/xharness/blob/3fc757d9b48e116713d1e0e3a3bd111a588905cb/src/Microsoft.DotNet.XHarness.iOS.Shared/SdkVersions.cs#L10
?

Reply:
Fixed!